libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/route/ReplayRouteSessionTest.kt

```diff
@@ -193,45 +198,34 @@ class ReplayRouteSessionTest {
             replayEventsObserver.captured.replayEvents(firstArg())
             replayer
         }
-        val routesUpdatedResult = mockActiveNavigationRoutes()
+        val routeProgress = mockRouteProgress()
 
         sut.setOptions(
             ReplayRouteSessionOptions.Builder()
                 .decodeMinDistance(0.001)
                 .build()
         )
         sut.onAttached(mapboxNavigation)
-        routesObserver.captured.onRoutesChanged(routesUpdatedResult)
+        routeProgressObserver.captured.onRouteProgressChanged(routeProgress)
 
         // Verify every point in the geometry was simulated
         val pushedPoints = pushedEvents.flatten().toList().map {
             val location = (it as ReplayEventUpdateLocation).location
             Point.fromLngLat(location.lon, location.lat)
         }
-        val geometry = routesUpdatedResult.navigationRoutes.first().directionsRoute.geometry()!!
+        val geometry = routeProgress.navigationRoute.directionsRoute.geometry()!!
         val geometryPoints = PolylineUtils.decode(geometry, 6)
-        assertTrue(pushedPoints.size > geometryPoints.size)
+        assertTrue(
+            "${pushedPoints.size} > ${geometryPoints.size}",
+            pushedPoints.size > geometryPoints.size
+        )
         assertTrue(
             geometryPoints.all { lhs ->
                 pushedPoints.firstOrNull { rhs -> lhs.equals(rhs) } != null
             }
         )
     }
 
-    @Test
-    fun `onAttached - should request gps location when resetLocationEnabled is true`() {
```

Comment:
Making this test pass also breaks the experience. The gps test can be considered covered by this test

`fun `onAttached - should push gps location when route is not set`() {`

But this will not be triggered until a [route is initialized](https://github.com/mapbox/mapbox-navigation-android/blob/0551bb3140e04a105bf87f25b46887129e280466/libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt#L136).